presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -192,16 +192,108 @@ only unsupported data types, Presto returns an error similar to the following ex
 
 ``Query 20231120_102910_00004_35dqb failed: Table 'public.unsupported_type_table' has no supported columns (all 1 columns are not supported).``
 
+
+SQL Support
+-----------
+
+The PostgreSQL connector allows querying and creating tables. Here are some examples of the SQL operations supported:
+
+ALTER TABLE
+^^^^^^^^^^^
+
+.. code-block:: sql
+
+    ALTER TABLE postgresql.public.sample_table ADD COLUMN new_col INT;
+    ALTER TABLE postgresql.public.sample_table DROP COLUMN new_col;
+    ALTER TABLE postgresql.public.sample_table RENAME COLUMN is_active TO is_enabled;
+    ALTER TABLE postgresql.public.sample_table RENAME TO renamed_table;
+
+CREATE TABLE
+^^^^^^^^^^^^
+
+Create a new table named ``test_integer`` in the ``public`` schema:
+
+.. code-block:: sql
+
+    CREATE TABLE postgresql.public.test_integer ( id INTEGER );
+
+CREATE TABLE AS SELECT
+^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new table ``new_table`` from an existing table ``renamed_table``:
+
+.. code-block:: sql
+
+    CREATE TABLE postgresql.public.new_table AS SELECT * FROM postgresql.public.renamed_table;
+
+DESCRIBE
+^^^^^^^^
+
+You can see a list of the columns in the ``clicks`` table in the ``web`` database
+using either of the following:
+
+.. code-block:: sql
+
+    DESCRIBE postgresql.web.clicks;
+    SHOW COLUMNS FROM postgresql.web.clicks;
+
+INSERT INTO
+^^^^^^^^^^^
+
+Insert data into the ``renamed_table`` table:
+
+.. code-block:: sql
+
+    INSERT INTO postgresql.public.renamed_table (id, name) VALUES (1, 'Test');
+
+SELECT
+^^^^^^
+
+.. code-block:: sql
+
+    SELECT * FROM postgresql.public.test_integer;
+
+SHOW SCHEMAS
+^^^^^^^^^^^^
+
+You can see the available PostgreSQL schemas by running ``SHOW SCHEMAS``:
+
+.. code-block:: sql
+
+    SHOW SCHEMAS FROM postgresql;
+
+SHOW TABLES
+^^^^^^^^^^^
+
+If you have a PostgreSQL schema named ``web``, you can view the tables
+in this schema by running ``SHOW TABLES``:
+
+.. code-block:: sql
+
+    SHOW TABLES FROM postgresql.web;
+
+TRUNCATE
+^^^^^^^^
+
+Delete all of the data from the table ``renamed_table`` without dropping the table:
+
+.. code-block:: sql
+
+    TRUNCATE TABLE postgresql.public.renamed_table;
+
 PostgreSQL Connector Limitations
 --------------------------------
 
-The following SQL statements are not yet supported:
+The following SQL statements are not supported:
 
+* :doc:`/sql/alter-table` (ALTER COLUMN TYPE)
```

Comment:
I am not sure I understand the meaning of `(ALTER COLUMN TYPE)` here. The capitalization suggests that when I follow the link to the ALTER TABLE page, I might find ALTER COLUMN TYPE but those words are not there. 
Can you explain?